scripts/extract-hvd-content.ts

```diff
@@ -46,16 +45,24 @@ export const HVD_CONTENT_DIR =
 
```

Comment:
On L41:
"Otherwise, use the path generated from the `extractHvdContent` script"
->
"Otherwise, use the path generated from the `extractINGHvdContent` script